src/pages/IndividualReport.vue

```diff
@@ -0,0 +1,410 @@
+    const imgData = canvas.toDataURL('image/jpeg', 0.7, { willReadFrequently: true });
+    const scaledCanvasHeight = canvas.height * returnScaleFactor(canvas.width);
+    // Add a new page for each task if there is no more space in the page for task desc and graph
+    if (yCounter + scaledCanvasHeight + offset > 287) {
+      document.addPage();
+      yCounter = 10;
+    } else {
+      // Add Margin
+      yCounter += 5;
+    }
+
+    document.addImage(imgData, 'JPEG', 10, yCounter, pageWidth, scaledCanvasHeight);
+    yCounter += scaledCanvasHeight;
+  });
+  return yCounter;
+};
+
+const exportToPdf = async () => {
+  exportLoading.value = true; // Set loading icon in button to prevent multiple clicks
+  if (!expanded.value) {
+    setExpand();
+  }
+  await new Promise((resolve) => setTimeout(resolve, 250));
+
+  const doc = new jsPDF();
+  let yCounter = 10; // yCounter tracks the y position in the PDF
+
+  // Add At a Glance Charts and report header to the PDF
+  const individualReport = document.getElementById('individual-report-export');
+  if (individualReport !== null) {
+    await addElementToPdf(individualReport, doc, yCounter);
+  }
+
+  doc.save(`IndividualScoreReport_${studentFirstName.value}${studentLastName.value}.pdf`),
+    (exportLoading.value = false);
+
+  return;
+};
+
+const tasks = computed(() => taskData?.value?.map((assignment) => assignment.taskId));
+
+const formattedTasks = computed(() => {
+  return (
+    tasks?.value
+      ?.map((taskId) => (extendedTaskTitle[taskId] ? extendedTaskTitle[taskId] : taskId))
+      .sort((a, b) => {
+        if (Object.keys(taskDisplayNames).includes(a.taskId) && Object.keys(taskDisplayNames).includes(b.taskId)) {
+          return taskDisplayNames[a.taskId].order - taskDisplayNames[b.taskId].order;
+        } else {
+          return -1;
+        }
+      })
+      .join(', ') + '.'
+  );
+});
+
+const studentFirstName = computed(() => {
+  if (studentData.value.name.first == undefined) return studentData.value.username;
+  return studentData.value.name.first;
+});
+
+const studentLastName = computed(() => {
+  if (!studentData.value.name) return '';
+  return studentData.value.name.last;
+});
+
+const extendedTaskTitle = {
+  swr: 'ROAR-Word',
+  'swr-es': 'ROAR-Palabra',
+  pa: 'ROAR-Phoneme',
+  sre: 'ROAR-Sentence',
+  vocab: 'ROAR-Picture Vocabulary ',
+  multichoice: 'ROAR-Multiple Choice',
+  morph: 'ROAR-Morphology',
+  cva: 'ROAR-Written Vocabulary',
+  letter: 'ROAR-Letter',
+  comp: 'ROAR-Comprehension',
+  syntax: 'ROAR-Syntax',
+  fluency: 'ROAM-Fluency',
+};
+
+function getGradeWithSuffix(grade) {
+  if (getGrade(grade) < 1) {
+    return grade;
+  } else if (getGrade(grade) === 1) {
+    return grade + 'st';
+  } else if (getGrade(grade) === 2) {
+    return grade + 'nd';
+  } else if (getGrade(grade) === 3) {
+    return grade + 'rd';
+  } else if (getGrade(grade) >= 4 && getGrade(grade) <= 12) {
```

Comment:
I think we technically allow up to grade 13 in the database.
```suggestion
  } else if (getGrade(grade) >= 4 && getGrade(grade) <= 13) {
```